.github/workflows/platform-content-sync.yml

```diff
@@ -0,0 +1,24 @@
+name: Sync content with platform
+
+on:
+  pull_request:
+    types:
+      - closed
+    paths:
+      - '**.mdx'
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: carbon-design-system/carbon-platform
+          ref: main
```

Comment:
```suggestion
```

@mattrosno This first job step should just be able to be removed entirely.

Reply:
Removing this would I think require the `gh issue create` command below to have a `--repo=carbon-design-system/carbon-platform` flag on the command.

Reply:
Yeah exactly. I just did the checkout so `gh` would have the proper context to open the issue on the right repo.

Using just a single command it's easy enough to use the `--repo` flag but didn't know if there were additional commands or things you all might want to add in addition to the issue being opened.

I'm good with either approach 👍

Reply:
@tay1orjones  I guess we have lots of available runners and actions minutes, so the time incurred by the checkout is probably not an issue 😅